internal/bminventory/inventory.go

```diff
@@ -192,7 +192,14 @@ const ignitionConfigFormat = `{
 	    "name": "root"
 	  },
 	  "append": [{ "source": "{{.ServiceIPs}}" }]
-  	}{{end}}]
+	}{{end}}],
+    "directories": [{
```

Comment:
Unsure this is required, I think the install command will generate this directory in case it doesn't exists